packages/truffle-core/lib/repl.js

```diff
@@ -74,6 +74,12 @@ ReplManager.prototype.start = function(options) {
     self.emit("exit");
   });
 
+  // Bubble the internal repl's reset event
+  this.repl.on("reset", function() {
+    process.stdout.write('\u001B[2J\u001B[0;0f');
```

Comment:
Does this only work for POSIX-compliant terminals? Do you know if this works on Windows as well?

Reply:
Hi @gnidan
Thanks for your reply, I'd test on azure window 10, it worked!
Looking forward this commit to be merged:)